src/core/database.py

```diff
@@ -104,7 +104,8 @@ def _createFieldsForEnvironmentTable(self) -> QgsFields:
         fields.append(self._createFieldShortText("glareSever"))
         fields.append(QgsField("cloudCover", QVariant.Int))
         fields.append(QgsField("visibility", QVariant.Int))
-        fields.append(QgsField("subjective", QVariant.String, len=1))
+        fields.append(QgsField("subjectiveMam", QVariant.String, len=2))
+        fields.append(QgsField("subjectiveBirds", QVariant.String, len=2))
```

Comment:
@pblottiere subjective field changed into two new fields that has been added in last excel file

Reply:
Ok :+1: